packages/core/test/predictableExec.test.ts

```diff
@@ -20,7 +22,9 @@ describe('predictableExec', () => {
             },
             assign(() => {
               actual.push('assign');
-              return {};
+              return {
+                assigned: true
+              };
```

Comment:
This isn't used by the test anyhow
```suggestion
              return {};
```